config/testSetup.ts

```diff
@@ -1,5 +1,7 @@
-import { configure } from "enzyme";
-import Adapter from "enzyme-adapter-react-16";
-import "@testing-library/jest-dom/extend-expect";
-
-configure({ adapter: new Adapter() });
+global.console = {
+    error: console.error,
+    info: console.info,
+    log: jest.fn(),
+    warn: jest.fn(),
+    debug: jest.fn(),
+};
```

Comment:
import { TextDecoder, TextEncoder } from 'util';

```suggestion
global.console = {
    error: console.error,
    info: console.info,
    log: jest.fn(),
    warn: jest.fn(),
    debug: jest.fn(),
};
```

// Polyfill for encoding which isn't present globally in jsdom
global.TextEncoder = TextEncoder
global.TextDecoder = TextDecoder